commit 99747dcef140fcb72575763545bc509fa88bea0a
Author: Sam Trenholme <sam.trenholme@gmail.com>
Date:   Sat Feb 8 23:07:29 2014 -0800

    Default random seed file is now secret.txt in Deadwood.exe compiled for Windows

diff --git a/deadwood-github/src/DwSys.c b/deadwood-github/src/DwSys.c
index 8f91640..8fd9836 100644
--- a/deadwood-github/src/DwSys.c
+++ b/deadwood-github/src/DwSys.c
@@ -574,7 +574,11 @@ void get_entropy_from_seedfile(uint8_t *noise,int len) {
         int seed = -1;
 
         if(key_s[DWM_S_random_seed_file] == 0) {
+#ifdef MINGW
+                filename = "secret.txt"; /* Default filename */
+#else /* MINGW */
                 filename = "/dev/urandom"; /* Default filename */
+#endif /* MINGW */
         } else {
                 filename = (char *)dw_to_cstr(key_s[DWM_S_random_seed_file]);
                 zap = 1;
